### ISSUE TITLE
Qt platform plugin "xcb" missing

### ISSUE THREAD

**Issue by olivierkes** [pyinstaller/pyinstaller#1906]
_Some_ user of our PyQt5 application reported the following error when running:

```
$ ./manuskript
This application failed to start because it could not find or load the Qt platform plugin "xcb".
Reinstalling the application may fix this problem.
Fatal Python error: Aborted

Current thread 0x00007fecb4108740 (most recent call first):
File "manuskript/main.py", line 18 in run
File "", line 13 in
Aborted
```

Package: https://github.com/olivierkes/manuskript/releases/download/0.2.0/manuskript-0.2.0-linux.zip

Packaged with: PyInstaller 3.1.1, Python 3.4.3, on Kubuntu 14.04 with Qt 5.2.1

Fails to run on: Netrunner 14.04.

**Comment by htgoebel** [comment]
Is this still an issue? If so, have you tried the current development version and is it solved there?

**Comment by eshizhan** [comment]
I tried the current development version, issue still here. But 3.2.1 is OK.

**Comment by htgoebel** [comment]
@eshizhan Thanks for this info. Can you please `git bisect` between 8998a9d544c5aa65e5479639b743b3c5dc1eebad (which is where `for-next-relase` was branched of) and develop to find out which commit caused the issue?

Adopt the following script (scratch!) to your needs, save it as `run-bisect.sh` and bisect `git bisect run sh ./run-bisect.sh` to automate the process:
```
pyinstaller --noconfirm my-app.py
dist/my-app/my-app
rc=$?
exit $rc
```

**Comment by eshizhan** [comment]
https://github.com/pyinstaller/pyinstaller/commit/675d66a21e8c5f7d1de538b05d8d2ee5e2871c51

**Comment by htgoebel** [comment]
Thanks you very much for bisecting :-)

**Comment by gaining** [comment]
Hello, I'm having the same issue, packaged on LinuxMint 18.3 but can't run on ubuntu 17.10 because of the "cannot find xcb library" issue.  Is there an easy work around? Maybe manually moving files around and then package?

**Comment by bjones1** [comment]
I've submitted a PR that hopefully fixes this -- see #3233. Try that and please leave feedback in there.
